01/NNRepos/wordvalue.py

```diff
@@ -0,0 +1,23 @@
+from data import DICTIONARY, LETTER_SCORES
+from typing import Optional, List
+
+
+def load_words() -> List[str]:
+    """Load dictionary into a list and return list"""
+    with open(DICTIONARY) as f:
+        return [line.strip() for line in f.readlines()]
+
+
+def calc_word_value(word: str) -> int:
+    """Calculate the value of the word entered into function
+    using imported constant mapping LETTER_SCORES"""
+    return sum(LETTER_SCORES[letter] for letter in list(word.replace('-', '').upper()))
```

Comment:
I don't think you need the `list()`, `word` is already iterable. The solution also uses `LETTER_SCORES.get(char.upper(), 0)` which is safer, because with `LETTER_SCORES[letter]` any unexpected letter would raise a `KeyError`.

Reply:
i actually used the `KeyError` for debugging (that's how i've discovered that `-` is a legal character), but for correctness, i guess you're right.